rslib/src/search/parser.rs

```diff
@@ -359,19 +367,31 @@ fn parse_rated(val: &str) -> ParseResult<SearchNode<'static>> {
 
     let ease = match it.next() {
         Some(v) => {
-            let n: u8 = v.parse()?;
-            if n < 5 {
-                Some(n)
+            let u: u8 = v.parse().unwrap();
```

Comment:
Unwrap should only ever be used when we're sure the value exists. Because of this change, rated:1:x will throw an exception, and would crash the program if pyo3 didn't catch panics.